pkg/standalone/standalone.go

```diff
@@ -612,41 +612,45 @@ func untar(filepath, targetDir, binaryFilePrefix string) (string, error) {
 
 	tr := tar.NewReader(gzr)
 
+	foundBinary := ""
 	for {
 		header, err := tr.Next()
 
-		switch {
-		case err == io.EOF:
-			return "", fmt.Errorf("file is empty")
-		case err != nil:
+		if err == io.EOF {
```

Comment:
We can still have switch block...no need to change to if/else blocks...

Reply:
Actually, it's kind of surprising but this does not work with the switch statement. the 'break' statement will break out of the switch statement rather than the for. My options were to either do this or create a label for the outer for loop and break out of that, and I went with this solution.